app/routes.py

```diff
@@ -48,14 +48,29 @@ def menu_browse():
     restaurants = utils.get_restaurant_details(res_ids)
 
     return render_template('menu-browse.html', restaurants=restaurants, isAdd=True)
-	
-@app.route('/menu-compare', methods=['GET'])
+
+@app.route('/menu-compare/', methods=['GET', 'POST'])
```

Comment:
should this be a GET and a POST, or just a POST?

Reply:
Yeah, it doesn't work without GET.